opendevin/controller/agent_controller.py

```diff
@@ -225,6 +225,11 @@ async def set_agent_state_to(self, new_state: AgentState):
         ):
             # user intends to interrupt traffic control and let the task resume temporarily
             self.state.traffic_control_state = TrafficControlState.PAUSED
+            # User has chosen to deliberately continue - lets double the max iterations
```

Comment:
This is de facto exponential growth. Maybe we should document it somewhere, and/or pop up a message on web GUI? This can be done in a follow-up issue/PR, though.

Reply:
Replaced the `*= 2` with adding the initial max_iterations the AgentController got passed into.